doc/classes/Input.xml

```diff
@@ -301,6 +301,7 @@
 				Input.ParseInputEvent(cancelEvent);
 				[/csharp]
 				[/codeblocks]
+				[b]Note:[/b] Calling this function has no influence on the operating system. So for example sending an [InputEventMouseMotion] will not move the OS mouse cursor to the specified position (use [method warp_mouse] instead) and sending [code]ctrl+shift+escape[/code] as [InputEventKey] won't open the OS task manager.
```

Comment:
I'd prefer an example that's not OS specific. Maybe Alt+F4? Also should be `[kbd]` markup for keys:
```suggestion
				[b]Note:[/b] Calling this function has no influence on the operating system. So for example sending an [InputEventMouseMotion] will not move the OS mouse cursor to the specified position (use [method warp_mouse] instead) and sending [kbd]Alt + F4[/kbd] as [InputEventKey] won't close the window.
```

Reply:
Alt + F4 might still be OS specific, not sure it works on macOS or if it needs to be Cmd + Q.
But at least it's a very well known hotkey so it might still be good enough.

Reply:
> not sure it works on macOS or if it needs to be Cmd + Q

It's `Cmd + Q`, `Alt(Option) + F4` is not a standard key and won't do anything.

Reply:
I guess `Alt/Cmd + Tab` for app switching is the most common and most similar OS shortcut (`Alt` and `Cmd` have a different key code, but physically it's the same key, next on the left of the space bar).

Reply:
```suggestion
				[b]Note:[/b] Calling this function has no influence on the operating system. So for example sending an [InputEventMouseMotion] will not move the OS mouse cursor to the specified position (use [method warp_mouse] instead) and sending [kbd]Alt/Cmd + Tab[/kbd] as [InputEventKey] won't toggle between active windows.
```

Reply:
Thanks for the suggestions, I have included them. I wasn't aware of [kbd].